integreat_cms/cms/constants/opening_hours.py

```diff
@@ -30,6 +31,6 @@
                 },
             },
         },
-        "required": ["allDay", "closed", "timeSlots"],
+        "required": ["allDay", "closed", "appointmentOnly", "timeSlots"],
```

Comment:
I just noticed that this change is probably not a good idea - old POIs can no longer be saved. Reverting this would also allow us to leave all the fixtures and expected tes outputs unchanged, significantly reducing the diff of this PR.

The only alternative I can think of is using some JS to **always** mark the Opening Hours as changed, which is not exactly great either.

Would appreciate some input.

Reply:
Hmm, I think we should keep this line, because it also helps the app parsing the information if they know the exact JSON schema that they receive...
So I'd propose to write a data migration to change the JSON of all existing POIs... sure, more work in the short term, but at least we have a clear database state.